src/libs/antares/study/parts/hydro/finallevelinflowsmodifyer.h

```diff
@@ -0,0 +1,110 @@
+/*
+** Copyright 2007-2023 RTE
+** Authors: Antares_Simulator Team
+**
+** This file is part of Antares_Simulator.
+**
+** Antares_Simulator is free software: you can redistribute it and/or modify
+** it under the terms of the GNU General Public License as published by
+** the Free Software Foundation, either version 3 of the License, or
+** (at your option) any later version.
+**
+** There are special exceptions to the terms and conditions of the
+** license as they are applied to this software. View the full text of
+** the exceptions in file COPYING.txt in the directory of this software
+** distribution
+**
+** Antares_Simulator is distributed in the hope that it will be useful,
+** but WITHOUT ANY WARRANTY; without even the implied warranty of
+** MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+** GNU General Public License for more details.
+**
+** You should have received a copy of the GNU General Public License
+** along with Antares_Simulator. If not, see <http://www.gnu.org/licenses/>.
+**
+** SPDX-License-Identifier: licenceRef-GPL3_WITH_RTE-Exceptions
+*/
+#ifndef __ANTARES_LIBS_STUDY_PARTS_HYDRO_FINAL_LEVEL_INFLOWS_MODIFYER_H__
+#define __ANTARES_LIBS_STUDY_PARTS_HYDRO_FINAL_LEVEL_INFLOWS_MODIFYER_H__
+
+#include "../../area/area.h"
+#include <antares/emergency.h>
+
+namespace Antares::Data
+{
+
+/*!
+ ** \brief Final Reservoir Level data for a single area
+ */
+class FinalLevelInflowsModifier
+{
+private:
+    // Final reservoir level runtime data
+
+    // Simulation Data
+    uint simEndDay;
+
+    // simulation year
+    // overwritten after each MC year
+    uint yearIndex;
+
+    // data per area
+    // data overwritten after each MC year
+    double initialReservoirLevel;
+    double finalReservoirLevel;
+    double deltaReservoirLevel;
+    double reservoirCapacity;
+    double lowLevelLastDay;
+    double highLevelLastDay;
+    int initReservoirLvlMonth;
+
+public:
+    Area* areaPtr;
```

Comment:
```suggestion
    Area* area;
```

Consider making area `private`